src/core/song.cpp

```diff
@@ -1380,6 +1387,10 @@ Song::FileType Song::FiletypeByExtension(const QString &ext) {
   if (ext.compare("spc"_L1, Qt::CaseInsensitive) == 0) return FileType::SPC;
   if (ext.compare("vgm"_L1, Qt::CaseInsensitive) == 0) return FileType::VGM;
 
+  // this can also be mp4/aac, etc, so commenting for now  --paper
```

Comment:
If the extension is the same, than it's not much we can do here anyway, so just remove the comments.